server/@types/manage-recalls-api/models/ApiRecallDocument.ts

```diff
@@ -34,6 +34,4 @@ export namespace ApiRecallDocument {
         REVOCATION_ORDER = 'REVOCATION_ORDER',
         UNCATEGORISED = 'UNCATEGORISED',
     }
```

Comment:
You mentioned you were going to reset this change, as it will be overwritten anyway by the next run of swagger-to-ts

Reply:
Looks like it didn't override it, I pushed change with fix